### DISCUSSION TITLE
am getting Row update failed error right [Sep 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6590] [Discord Threads]
> 🙋‍♂️ **kannan6510**  
## am getting Row update failed error , right now we cannot able to enter anything in that field, if we entered am getting that error for all rows am getting this  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1146017440690610196/image.png?ex=652e5769&is=651be269&hm=1c92543f752f45a87e121fc46340e54ed0c3ec23e430e357183f5bdb01b39a76&)

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### can you refresh the page to see if someone delete the column?  
#  
> 👨‍🚀 **wingkwong**  
#### also u may check if the schema got changed  
#  
> 🙋‍♂️ **kannan6510**  
#### NO Rows are all avilable  
#  
> 👨‍🚀 **wingkwong**  
#### i mean columns, not rows  
#  
> 👨‍🚀 **wingkwong**  
#### also can you share the error logs  
#  
> 🙋‍♂️ **kannan6510**  
#### in that sheet if i entered in any cell am getting this error  
#  
> 🙋‍♂️ **kannan6510**  
#### How can i collect error log  
#  
> 👨‍🚀 **wingkwong**  
#### u using docker?  
#  
> 👨‍🚀 **wingkwong**  
#### i mean you may check your db  
#  
> 🙋‍♂️ **kannan6510**  
#### no in web am using  
#  
> 👨‍🚀 **wingkwong**  
#### lemme put in this way. please share your project info by following    
#### https://docs.nocodb.com/FAQs#how-to-check-my-project-info-  
#  
> 🙋‍♂️ **kannan6510**  
#### Node: **v16.20.1**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### RootDB: **pg**   
#### PackageVersion: **0.111.0**  
#  
> 👨‍🚀 **wingkwong**  
#### ok u r using docker  
#  
> 👨‍🚀 **wingkwong**  
#### did u setup nocodb?  
#  
> 👨‍🚀 **wingkwong**  
#### or u r just a user?  
#  
> 🙋‍♂️ **kannan6510**  
#### am a user  
#  
> 🙋‍♂️ **kannan6510**  
#### i will confirm once with my team  
#  
> 🙋‍♂️ **kannan6510**  
#### unable to upoad the text file  
#  
> 👨‍🚀 **wingkwong**  
#### you may ask the dev to check the docker logs to retrieve the error msgs  
#  
> 🙋‍♂️ **kannan6510**  
#### i got it  
#  
> 🙋‍♂️ **kannan6510**  
####   
[message.txt](https://cdn.discordapp.com/attachments/1146017440933875762/1146029660023558164/message.txt?ex=652e62cb&is=651bedcb&hm=396da712609e780a8a0e0fdcba639402c4cef95bb54563a0e072f0a918e649bb&)  
#  
> 🙋‍♂️ **kannan6510**  
#### error: update "nc_4vwu__Ops - Chapter Status" set "Printing Status" = NULL where "id" = '1' - record "new" has no field "Courier Tracking Id"  
#  
> 🙋‍♂️ **kannan6510**  
#### once i add the couier tracking id i can able to update  
#  
> 🙋‍♂️ **kannan6510**  
####   
![image0](https://cdn.discordapp.com/attachments/1146017440933875762/1146030752371327047/image.png?ex=652e63cf&is=651beecf&hm=eac91f258a8a8fd6b86d4fac183a918e713c9ba9031c5283bc113869fe66d7ed&)  
#  
> 🙋‍♂️ **kannan6510**  
#### whats the cause of adding date and time in that column  
#  
> 👨‍🚀 **wingkwong**  
#### i think your column has the default value `NOW()`  
#  
> 👨‍🚀 **wingkwong**  
#### you may ask the dev to check the DB  
#  
> 🙋‍♂️ **kannan6510**  
#### okay , Thanks for your Support and enlighten me for how to do cross verify and check for issue. in future if anything like appear sure this will guide me  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1146017440933875762)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
